### PR TITLE
Better errors

### PR DESCRIPTION
**Pull request by FFY00**
This could have helped spot #716/#834 much easier.

### CODE REVIEW

**Comment by FFY00** [comment]
Rebased on top of #890

File: ratbagd/ratbagd-device.c [hunk 1]
```diff
@@ -172,6 +172,8 @@ static void ratbagd_device_commit_pending(void *data)
 	int r;
 
 	r = ratbag_device_commit(device->lib_device);
+	if (r)
+		log_error("error committing device (%d)\n", r);
```
Comment (stephanlachnit) [inline ratbagd/ratbagd-device.c]: Shouldn't it be `log_error(ctx,"...")`?

Comment (FFY00) [reply ratbagd/ratbagd-device.c]: Ah, yeah. Thanks for catching that.

Comment (FFY00) [reply ratbagd/ratbagd-device.c]: Actually no, not here. I just checked and we only use that for the drivers.

Comment (stephanlachnit) [reply ratbagd/ratbagd-device.c]: Whoops yes, think I mixed something up.

**Pull request merged**